rendering_resource_manager_service/deployment/rrm/populateRRM.txt

```diff
@@ -2,10 +2,16 @@
 # of the rendering resource manager.
 
 # Livre
```

Comment:
Do you think you can split the --data option with some quoting to make is span multiple lines? That will help spot differences if you make changes later.